src/alpine/3.16/helix/amd64/Dockerfile

```diff
@@ -69,10 +69,10 @@ RUN apk update && apk add --no-cache && \
         openssl-dev \
         perl && \
     cd /tmp && \
-    git clone --depth 1 --single-branch --branch main --recursive https://github.com/dotnet/msquic && \
-    cd msquic/src/msquic && \
+    git clone --depth 1 --single-branch --branch main --recursive https://github.com/microsoft/msquic && \
```

Comment:
One option here is to track MsQuic releases. That would match other platforms where we consume published binaries. 
The `dotnet/msquic` give us some control in the past.  
Pointing at `main` may cause some instabilities and surprises but on the other hand it allows us to see issues sonner.

Reply:
@CarnaViire suggested the same thing, I'll change it to get this in.